specs/devexperience/DevExperience.md

```diff
@@ -0,0 +1,181 @@
+
+#### 5.2.3 - Choosing your Windows APIs
+The UWP target platform selector is problematic with Reunion. The Reunion APIs are designed to ship out of band with the OS, and with this, users are able to select Windows versions that are not compatible with their Reunion packages. This UI should just be disabled, since we hard code the TargetPlatformVersion (TPV) in the project file to match what reunion supports. The default TargetPlatformMinVersion (TPMV) if not specified, will match the TPV, and developers can change it by setting the TPMV property. When migrating existing UWP apps, they will likely have TPMV set, and the existing property will be respected.
+
+### 5.3 – MSIX Packaging
+This section will go into more detail for the MSIX packaging story, and the steps that users will take in Visual Studio, or from the command line, in order to successfully build MSIX packages.
+
+#### 5.3.1 – Single-Exe Packages
+Single-exe packages are the default, and like UWP projects today, won't require the Windows Application Packaging (WAP) project. Developers will be able to build and debug these apps directly in Visual Studio.
+
+#### 5.3.2 - Multi-Exe packages
+Many apps, like the Windows terminal, package multiple exes into their msix. For this reason, the .wapproj will not be fully retired. However, this project can solely be used for packaging their app for distribution, and not get in the way of their inner-loop.
+
+#### 5.3.3 - Creating App Bundles and Publishing to the Microsoft Store
+In this section, we’ll be using command line interface (cli) examples for simplicity and greater articulation of how the technology works. While all these actions are accomplishable through the VS UI, developers still require cli workflows in their day-to-day, and especially for Continuous Integration (CI) environments. The VS UI will be a nice wrapper around these tasks.
+
+Today, in order to build app bundles from the cli, there is a complicated incantation that you need to specify on the command line:
+
+>msbuild WindowXamlAppCs.sln /m /p:Platform=x64 /p:UapAppxPackageBuildMode=StoreOnly /p:AppxBundle=Always /p:AppxBundlePlatforms="x64|arm64" /p:Configuration=Release
+
+This invokes MSBuild in a recursive manner which rebuilds the project for multiple architectures. This behavior adds enormous complexity to the build and is difficult to maintain. Furthermore, a complex build is a slow build, and so this behavior *will not* be carried forward. Instead, we will only build individual .msix files per architecture, and we will enable developers to submit their apps to the store either by using the [Windows Store Azure DevOps task](https://marketplace.visualstudio.com/items?itemName=MS-RDX-MRO.windows-store-publish), and the VS `Publish->Create App Packages` UI will be updated to handle individual .msix files.
```

Comment:
> ...and the VS `Publish->Create App Packages` UI will be updated to handle individual .msix files.

By 'individual .msix files' you mean not bundles?

Are you aware of [Flat bundles](https://docs.microsoft.com/windows/msix/package/flat-bundles)? Will that be supported?

Reply:
Yes, at least not the bundles that contain multiple arch's. I was not aware of flat bundles, but that does look *super* promising! What's the workflow for creating a flat bundle? Do I need to author a special manifest?

Reply:
@stevenbrix here's more info on [flat bundles](https://docs.microsoft.com/windows/msix/package/flat-bundles). The key difference is running `makeappx.exe` with a different command line.